application/common/mail/reset/on-behalf.php

```diff
@@ -0,0 +1,15 @@
+Hi there,
+
+<?php echo \yii\helpers\Html::encode($name); ?> recently requested a password change for their
+<?php echo \Yii::$app->params['appName']; ?> account and they have requested your assistance.
```

Comment:
You're right, I already decided that needed to be renamed to make more sense, I'm, thinking something like ```accountName``` instead